week-2/Homework/mandatory/3-playing-computer.js

```diff
@@ -40,3 +41,11 @@ for (let i = 0; i < 5; ++i) {
     console.log(e);
   }
 }
+// there is no variable b declaration, i would add it to escape error,
+//but it is required to remove a line in this case i would also remove a declaration
+//and pass both variables to the function as parameters. i wrote this and then understood thatthisprogram is not only a function.
+//I have the only solution to declare a and b to run it.
+// i think f2 is called 3 times because 0 gives 0 remainder when devided by 2
+//at the same time f1 is called in 2 cases when i=1 and i= 3.
+//I confirmed it by counting logged lines 3 logs for tree log lines before calling loop with f functions and 5
+//for 3X*2+2*f1 calls.
```

Comment:
This looks great :) Can you add your answers to questions 6 and 7 too?

Reply:
i will do new pull request now.